src/components/feedback/Feedback.tsx

```diff
@@ -73,94 +70,105 @@ export const Feedback: FC = () => {
 
   const sendForm = (e: MouseEvent) => {
     e.preventDefault();
-    setInLocalStorage(state, FEEDBACK_DATA_LS_LABEL);
+    sendFeedback(state);
+    setUpdatedProp(Math.random());
     resetForm();
   };
 
-  const ratingColor = useMemo(() => {
-    const { rating } = state;
-    if (rating >= 8) return 'green';
-    if (rating >= 5) return 'yellow';
-    if (rating < 5) return 'red';
-  }, [state.rating]);
-
   return (
     <main className={commonStyle.container}>
       <h2 className={styles['feedback-title']}>Please, send feedback</h2>
       <form className={styles['feedback-form']}>
-        <div className={styles['feedback-required-data']}>
-          <div className={styles['feedback-required-data__group']}>
-            <label htmlFor="feedback-name" className={styles['feedback-input-label']}>
-              What is your name?
-            </label>
-            <input
-              type="text"
-              id="feedback-name"
-              name="name"
-              value={state.name}
-              className={styles['feedback-input-text']}
-              onChange={validatedInputHandler}
-              required
-            />
-            {!name.isValid && state.name.length ? (
-              <span className={styles['validation-error']}>Incorrect name</span>
-            ) : null}
-          </div>
-          <div className={styles['feedback-required-data__group']}>
-            <label htmlFor="feedback-email" className={styles['feedback-input-label']}>
-              Enter your email
-            </label>
-            <input
-              type="email"
-              id="feedback-email"
-              name="email"
-              value={state.email}
-              className={styles['feedback-input-text']}
-              onChange={validatedInputHandler}
-              required
-            />
-            {!email.isValid && state.email.length ? (
-              <span className={styles['validation-error']}>Incorrect email</span>
-            ) : null}
-          </div>
+        <div className={styles['feedback-user-data__group']}>
+          <label htmlFor="feedback-name" className={styles['feedback-input-label']}>
+            What is your name?
+          </label>
+          <input
+            type="text"
+            id="feedback-name"
+            name="name"
+            value={state.name}
+            className={styles['feedback-input']}
+            onChange={validatedInputHandler}
+            placeholder="required field"
+            required
+          />
+          {!isValid.name && state.name.length ? (
+            <span className={styles['validation-error']}>
+              The name must include letters and numbers and be longer than 4 characters
+            </span>
+          ) : null}
```

Comment:
no need to use ternary operator here, just use `&&`. also no need to write `.length`, `!!state.name` is enough